hack/cherry-pick.sh

```diff
@@ -95,7 +92,7 @@ if ! git log -n1 --format=%H "${BRANCH}" >/dev/null 2>&1; then
   exit 1
 fi
 
-NEWBRANCHREQ="automated-cherry-pick-of-${PULLDASH}" # "Required" portion for tools.
+NEWBRANCHREQ="automated-cherry-pick-of-#${PULL}" # "Required" portion for tools.
```

Comment:
We have some git branching logic for building server/webapp/enterprise repo together. The naming of the cherry-pick branches could collide and there is also no cleanup in the repository of the cherry-picking bot, so it could happen that outdated server branch with the equally named webapp cherry pick pull request collides and then it is confusing why the pipeline fails. one solution would be to add somehow the project name into the branch name. Maybe as an acronym.

Reply:
I've added the repo name to the branch